lib/fog/azurerm/docs/storage.md

```diff
@@ -163,122 +163,158 @@ azure_storage_service.delete_disk('<Data Disk Name>')
 Create a storage container in the current storage account.
 
 ```ruby
-container = azure_storage_service.create_container(
-  name: '<container name>'
+directory = azure_storage_service.directories.create(
+  key: '<container name>',
+  public: true
 )
-puts "#{container.name}"
+puts directory.key
 ```
 
 ## List storage containers
 
 List all the storage containers in the current storage accounts.
 
 ```ruby
-azure_storage_service.directories.each do |directory|
-  puts "#{directory.name}"
+azure_storage_service.directories.all.each do |directory|
```

Comment:
This is an example for how to list all containers in the storage account.
directories is a Fog collection but do not execute anything directly.